routes/category.js

```diff
@@ -34,6 +34,7 @@ router.get(
 router.use('/:categoryId?/subjects/:subjectId?/offers', isEntityValid(nestedParam), offerRouter)
 router.use('/:id?/subjects', isEntityValid(param), subjectRouter)
 router.get('/', asyncWrapper(categoryController.getCategories))
+router.post('/', asyncWrapper(categoryController.addCategory))
```

Comment:
i suppose this route have to be restricted only for admins

Reply:
in the future, user will send request to create new category/subject and then admin will accept (or not) this request. For now, user can add category without admin`s approve

Reply:
oh, okey i got it